### PR TITLE
Hardcode monitoring host to 1

### PR DESCRIPTION
**Pull request by MalloZup**
fix issue #193

I will test this pr

### CODE REVIEW

**Comment by MalloZup** [comment]
tested on libvirt , worked fine!

File: libvirt/modules/monitoring/main.tf [hunk 1]
```diff
@@ -37,7 +35,7 @@ resource "libvirt_domain" "monitoring_domain" {
   network_interface {
     wait_for_lease = false
     network_id     = var.network_id
-    hostname       = "${var.name}${var.monitoring_count > 1 ? "0${count.index + 1}" : ""}"
+    hostname       = "${var.name}"
```
Comment (arbulu89) [inline libvirt/modules/monitoring/main.tf]: I will add the `workspace` before the name as we are doing in the `grains` files

Comment (MalloZup) [reply libvirt/modules/monitoring/main.tf]: ok thx

**Comment by MalloZup** [comment]
@arbulu89 thx adressed

**Comment by MalloZup** [comment]
@arbulu89 ok I have merged.. I will offer you a cofee next time we see in person

**Pull request merged**